tests/nemo_text_processing/es/test_cardinal.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from nemo_text_processing.inverse_text_normalization.inverse_normalize import InverseNormalizer
+from nemo_text_processing.text_normalization.normalize import Normalizer
+from nemo_text_processing.text_normalization.normalize_with_audio import NormalizerWithAudio
+from parameterized import parameterized
+
+from ..utils import PYNINI_AVAILABLE, parse_test_case_file
+
+
+class TestCardinal:
+    inverse_normalizer_es = InverseNormalizer(lang='es') if PYNINI_AVAILABLE else None
```

Comment:
could you add cache dir? Same for other files?
like here: https://github.com/NVIDIA/NeMo/blob/main/tests/nemo_text_processing/de/test_date.py#L24